foxglove/cmd/export.go

```diff
@@ -796,8 +796,11 @@ func executeExport(
 		bearerToken,
 		userAgent,
 	)
-	progressWriter := progressbar.DefaultBytes(-1, "exporting")
-	writer := io.MultiWriter(w, progressWriter)
+	writer := w
+	if request.OutputFormat != "json" {
```

Comment:
It seems like the better check here would be `if stdoutRedirected()`; our default json behavior today is more of a side-effect.

Reply:
And it would be nice to still have a progress bar when redirecting json output.

Reply:
good suggestion, done